tests/providers/jersey-tides.test.ts

```
@@ -0,0 +1,262 @@
+  });
+
+  // Clear the module cache to pick up new env vars
+  vi.resetModules();
+  const { jerseyTideProvider: freshProvider } = await import("../../src/providers/jersey-tides");
+  
+  await freshProvider.getEvents(1);
+  
+  const callArgs = mockFetch.mock.calls[0];
+  expect(callArgs[0]).toContain("&datum=MLLW");
+  
+  delete process.env.STORM_DATUM;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)